task_01/ex5.py

```diff
@@ -0,0 +1,12 @@
+"""Functions."""
+
+
+def greet(name):
+    """Print a greeting to an argument."""
+    print('Hello,', name)
```

Comment:
It's a tricky moment, but we should care where we should put spaces. 
I think we want to read something like 

>  Hello, Maxim
>
rather than 
> Hello,Maxim
> 
Usually, we add space to constant string.

Reply:
print adds space here.

Reply:
yeah, my fault